elektronLogging/createLogDir.go

```diff
@@ -0,0 +1,34 @@
+package elektronLogging
+
+import (
+	logrus "github.com/sirupsen/logrus"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var logDir string
+
+func GetLogDir(startTime time.Time, prefix string) {
+	if logDir == "" {
+		logDir = createLogDir(prefix, startTime)
+	}
+}
+
+func createLogDir(prefix string, startTime time.Time) string {
+
+	// Creating directory to store all logs for this run
```

Comment:
Extend the comment to show how the format of the log directory name is going to look.

Reply:
Done